modern/clang/Dockerfile

```diff
@@ -155,4 +155,4 @@ RUN sudo mv /tmp/gcc/lib64 /usr/local/ \
     && sudo update-alternatives --install /usr/local/bin/ld ld /usr/local/bin/ld.lld 100 \
     && sudo rm /etc/ld.so.cache \
     && sudo ldconfig -C /etc/ld.so.cache \
-    && conan config init --force
```

Comment:
Why not libstdc++11 for clang too?

Reply:
Or `libc++`? I have doubts, but for GCC I think there are no doubts, right?

Reply:
libstdc++11 I think will work fine, but libc++ has some limitations. We saw some projects which didn't build with libc++. Personally, I prefer libc++, but I would like to keep align/prepared for Conan 2.0 profiles.

Reply:
Conan 2.0 is not going to autodetect anything, so that won't be a problem.

The issue I try to fix with GCC is the getting-started. I'm sure you have come across the ABI link error when using Conan many times... and you spend some time until you realize it is using _the wrong_ libcxx configuration. And it is wrong because the world already uses new ABI (the underlying system, binaries you can get from internet,...)

TBH, I don't use Clang, so I don't come across any error 🤷  But, I would say that `libc++` is more like the default for clang, so it would be expected if you start from scratch.

Reply:
> I'm sure you have come across the ABI link error when using Conan many times

True, that's why I use libstdc++11 by default now.

Let's update gcc for now and think about clang for a next clang.